### ISSUE TITLE
some psp tar.gz lib in html format

### ISSUE THREAD

**Issue by yoyz** [pspdev/psptoolchain#84]
$ cd build/psplibraries/
$ ./libraries.sh SDL_image

--2017-09-26 18:01:44--  http://sourceforge.net/projects/libpng/files/libpng15/older-releases/1.5.7/libpng-1.5.7.tar.gz
Résolution de sourceforge.net (sourceforge.net)... 52.84.125.228, 52.84.125.204, 52.84.125.56, ...
Connexion vers sourceforge.net (sourceforge.net)|52.84.125.228|:80...connecté.
requête HTTP transmise, en attente de la réponse...302 Moved Temporarily
Emplacement: https://sourceforge.net/#!/projects/libpng/files/libpng15/older-releases/1.5.7/libpng-1.5.7.tar.gz [suivant]
--2017-09-26 18:01:45--  https://sourceforge.net/  
Connexion vers sourceforge.net (sourceforge.net)|52.84.125.228|:443...connecté.
requête HTTP transmise, en attente de la réponse...200 OK
Longueur: 652 [text/html]
Sauvegarde en : «libpng-1.5.7.tar.gz»

100%[=============================================================================================================================>] 652         --.-K/s   ds 0s

2017-09-26 18:01:45 (64,5 MB/s) - «libpng-1.5.7.tar.gz» sauvegardé [652/652]

Extracting libpng-1.5.7.tar.gz...

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Couldn't install dependency libpng, aborting.
Failed installing SDL_image!
Installation finished.
Failed installing: SDL_image

$ file build/libpng-1.5.7.tar.gz 
build/libpng-1.5.7.tar.gz: HTML document, ASCII text

$ cat build/libpng-1.5.7.tar.gz 

<html><head>
<title>SourceForge</title>

<script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
<script src="https://sourceforge.net/js/mirrors.js"></script>
<script src="/js/sf.js"></script>
<script>
var DR_loc = DR_parse_hash_url();
if (DR_loc) {
    DR_sf_main(DR_loc);
} else {
    window.location.href = 'https://sourceforge.net/home.html';
}
</script>
</head><body>
<noscript>
We're sorry -- the Sourceforge site is currently in Disaster Recovery mode, and currently requires
the use of javascript to function.  Please check back later.
</noscript>
</body></html>

Solution at the root of pspdev, 
$ cd build/psplibraries/
$ wget https://zlib.net/fossils/zlib-1.2.8.tar.gz
$ mv zlib-1.2.8.tar.gz build/zlib-1.2.8.tar.gz
$ ./libraries.sh zlib

Hope it help

**Comment by carstene1ns** [comment]
Well, that was a temporary sf.net failure, so redownloading the file is fine in this case. It seems your output is unrelated (zlib vs. libpng) or maybe both failed to download.
However, we should probably check the downloaded files with a hash to be notified of such things.

**Comment by yoyz** [comment]
Yes, it seem to be a temporary source forge failure. I open this issue, to describe how to relaunch a given step for the library.
I don't know if this part is describe somewhere.
And, yes, a hash could be great.

**Comment by carstene1ns** [comment]
This is fixed when pacman will land. Moving this to psplibraries, we need a different solution for toolchain files.
